eng/pipelines/repoman.yml

```diff
@@ -96,19 +96,19 @@ extends:
                   Write-Host "##vso[task.setvariable variable=CommentBodyFile]$file"
                 displayName: Set Repoman Content
 
-              - task: PowerShell@2
-                displayName: Post PR comment
-                inputs:
-                  pwsh: true
-                  targetType: filePath
-                  filePath: ./eng/scripts/Update-PRComment.ps1
-                  arguments: >-
-                    -Repo 'azure/azure-dev'
-                    -PRNumber '$(PRNumber)'
-                    -BodyFile '$(CommentBodyFile)'
-                    -Tag '<!-- #comment-repoman-generate -->'
-                env:
-                  GH_TOKEN: $(azuresdk-github-pat)
+              # - task: PowerShell@2
```

Comment:
Just double checking here: we'll want to re-enable this before the final merge?

Reply:
Just did it now.  Will see if it works now.  If not, I might just override the checks based on the previous succesful run